### ISSUE TITLE
Javazon swap titans

### ISSUE THREAD

**Issue by alpharuns** [kolton/d2bot-with-kolbot#2048]
I was thinking of removing the lightning fury delay on the javazon and having an extra titans on swap. I can't seem to find such a function in the amazon config, does anyone have an idea how to get it to recognise how many javs left and then swap?

**Comment by mikedorsi** [comment]
I don't believe there is a way to track how many javs are left in the stack.  But it possible to make your character swap weapons when it goes to jab instead of throw.
